management-gateway/src/main/java/org/frankframework/management/bus/message/EmptyResponseMessage.java

```diff
@@ -13,9 +13,9 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 */
-package org.frankframework.management.bus;
+package org.frankframework.management.bus.message;
 
-public class EmptyResponseMessage extends StringResponseMessage {
+public class EmptyResponseMessage extends StringMessage {
```

Comment:
```suggestion
public class EmptyMessage extends StringMessage {
```
Or is this one always a response?

Reply:
Yea sorry, forgot to rename that one...